.github/workflows/lint-with-vale.yml

```
@@ -0,0 +1,21 @@
+---
+name: Linting with Vale on pull request
```

```suggestion
name: linter
```

---

I do not recommend this because it is possible to set up multiple linters. Specific job names make debugging easier.

---

Right now the check shows up as `Linting with Vale on pull request / Linting with Vale (pull_request)`. That's composed of:
* `Linting with Vale on pull request` (this line)
* `Linting with Vale` (name on line 11)
* `pull_request` (trigger, line 4)

You can see there's a lot of  duplication.

I'd still consider simplifying to:
```suggestion
name: Vale
```

And then make line 11 simply `Lint`. The result will be `Vale / Lint (pull_request)`. On the checks tab all jobs will be grouped under `Vale` as well (though there is only 1 job now).

You can also consider renaming the file to `vale.yml`. IMHO the `lint-with` is redundant and you can simply make the whole file just for all actions that Vale can do.

---

Done